src/Exception/KlaviyoApiException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Klaviyo\Exception;
+
+class KlaviyoApiException extends KlaviyoException {
+
+  public function __construct($message, $code, $previous = null) {
+        $message = 'Status Code ' . $code . ' - ' . $message;
```

Comment:
I think it would be fine to leave the message and status code separate (i.e. i don't think we even need to define the constructor in this class) as long as we're providing the message when we raise the KlaviyoApiException in the handleResponse method.